src/main.js

```diff
@@ -139,6 +139,18 @@ export class Draggable {
         };
     }
 
+    cancelDrag() {
+        unbind(this.document, "pointermove", this._pointermove);
+        unbind(this.document, "pointerup", this._pointerup);
+        unbind(this.document, "pointercancel", this._pointerup);
+
+        if (this.clickMoveClick) {
+            unbind(this.document, "contextmenu", preventDefault);
+        }
+
+        this._isDragging = false;
```

Comment:
We do not have such a flag used anywhere else - should this be added in the `onDrag` event handler, so that it has the right status, i.e. if you start the dragging action and you never call `cancelDrag`, it seems logical to have the flag set to `true`.

On the other hand - is there a need for such a flag? If you cancel the drag, then the events are not raised anymore, so you do not have a way to check whether dragging is still happening (not executing the event handlers is a clear indication that dragging is not performed) - and if you are getting the `onDrag` event handlers, then it is clear that dragging is happening.

Not sure about the original intention or whether we need to track whether `onDrag` has been canceled.